app/src/main/java/de/tum/in/tumcampusapp/component/tumui/calendar/CalendarActivity.java

```diff
@@ -89,13 +94,22 @@ public CalendarActivity() {
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
 
-        // Get a reference for the week view in the layout.
         mWeekView = findViewById(R.id.weekView);
 
         // The week view has infinite scrolling horizontally. We have to provide the events of a
         // month every time the month changes on the week view.
         mWeekView.setMonthChangeListener(this);
         mWeekView.setOnEventClickListener(this);
+        mWeekView.setScrollListener((newFirstVisibleDay, oldFirstVisibleDay) -> {
+            DateTime visibleDay = new LocalDate(newFirstVisibleDay).toDateTimeAtStartOfDay();
+            DateTime today = DateTime.now().withTimeAtStartOfDay();
+            boolean isToday = visibleDay.getMillis() == today.getMillis();
```

Comment:
Comparing millis is really tricky. Since we're using Joda, please just use the `.equals()`
Also `LocalDate.now()` gives today as a date, so we don't need to call `withTimeAtStartOfDay()`

I think I implemented this properly in `CafeteriaWithMenus`, but since this class is not Kotlin, we can't really reuse the extension function...